hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopFsGenerator.java

```diff
@@ -87,6 +87,10 @@ public Void call() throws Exception {
 
     configuration = createOzoneConfiguration();
     uri = URI.create(rootPath);
+    String scheme = uri.getScheme();
+    if (scheme == null) {
+      throw new IllegalArgumentException("--path requires FQDN");
+    }
```

Comment:
Shouldn't we use the default FS in that case? If no FS is specified explicitly, the defaultFs should be used, right? Something like this:
```
    if (scheme == null) {
      URI defaultUri = FileSystem.getDefaultUri(configuration)
      uri = new URI(defaultUri.getScheme(), defaultUri.getAuthority(), uri.getPath());
    }
```